packages/nextjs/app/builders/0x2d90C8bE0Df1BA58a66282bc4Ed03b330eBD7911/page.tsx

```diff
@@ -0,0 +1,201 @@
+                                  border-4 border-white dark:border-gray-800
+                                  transition-colors duration-300"
+                    >
+                      <Image
+                        src="/builders/oliveCartoon.webp"
+                        alt="Joseph Ochiagha"
+                        width={128}
+                        height={128}
+                        className="object-cover"
+                        priority
+                      />
+                    </div>
+                  </div>
+                </div>
+
+                {/* Name and bio with improved typography for both modes */}
+                <div className="mt-6 text-center">
+                  <h1 className="text-2xl font-bold text-gray-800 dark:text-gray-100 transition-colors duration-300">
+                    Joseph Ochiagha
+                  </h1>
+
+                  <div className="mt-2 flex items-center justify-center relative z-50">
+                    <div className="py-1 px-2 rounded-lg bg-blue-500 dark:bg-blue-900/40 z-20 relative shadow-sm border border-blue-200 dark:border-blue-800">
+                      <div className="text-white dark:text-blue-100">
+                        <Address address="0x2d90C8bE0Df1BA58a66282bc4Ed03b330eBD7911" format="short" size="sm" />
+                      </div>
+                    </div>
+                  </div>
+
+                  <p className="mt-3 text-base font-medium text-blue-600 dark:text-blue-400 transition-colors duration-300">
+                    Smart Contract Dev, In love with Cryptography
+                  </p>
+
+                  <p className="mt-2 text-sm max-w-sm mx-auto text-gray-600 dark:text-gray-300 transition-colors duration-300">
+                    Exploring the boundaries of blockchain technology and passionate about building secure, efficient
+                    smart contracts.
+                  </p>
+                </div>
+
+                {/* Skills chips with improved UI for both modes */}
+                <div className="mt-6">
+                  <div className="flex items-center gap-2 mb-3">
+                    <FaCode size="16px" className="text-blue-600 dark:text-blue-400 transition-colors duration-300" />
+                    <h2 className="text-base font-semibold text-gray-800 dark:text-gray-200 transition-colors duration-300">
+                      Skills
+                    </h2>
+                  </div>
+                  <div className="flex flex-wrap gap-2 justify-center">
+                    {skills.map(skill => (
+                      <span
+                        key={skill}
+                        className="px-3 py-1 text-sm rounded-full 
+                                bg-blue-50 text-blue-700 border border-blue-100 
+                                dark:bg-gray-700 dark:text-blue-300 dark:border-gray-600 
+                                transition-all duration-300 hover:bg-blue-100 dark:hover:bg-gray-600"
```

Comment:
You should only have hover effects for interactive elements. This hover effect misleads the user into thinking they can click it to navigate.